src/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './common';
```

Comment:
switching to manual generation?

Reply:
yeah. After I migrated the jest configuration from a .js to a .ts file, `ctix` started including it as a part of the indexing, and doesn't seem to obey its own 'exclude' directives, so I dropped that build step and just checked in these generated files. 

I can revisit this once I start working on the other box implementations, but I don't imagine that these index files are going to be changing much until then.